src/document/UIStateStore.tsx

```diff
@@ -273,7 +273,11 @@ export const UIStateStore = types
       types.union(
         ...Object.keys(PathGradients).map((key) => types.literal(key))
       )
-    )
+    ),
```

Comment:
This would be a good chance to make a sub-store ContextMenuStore, for organizational purposes. Not sure if I need that changed now though.

Reply:
I can put that as code base quality issue for future.